### PR TITLE
feat: add transaction item comp and show recent transactions on home page

### PR DESCRIPTION
**Pull request by Siqill**
### Description

Please provide a brief description of the changes you have made in your PR.

### Type of change

- [x] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [ ] Refactoring (doesn't change functionality, only the code)
- [ ] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)
- [x] Chakra components/constants/hooks have been used wherever possible

> If not explain here

- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change
homepage before

Add screenshots from the [current deployed state](https://sandbox-money-app.herokuapp.com/) of the changed components here.

#### Screenshots/Video after change

Add screenshots with the changes applied here.

https://user-images.githubusercontent.com/44428466/182486481-4e2d3ee4-cf01-42db-aa11-3ab4a1966200.mp4

### CODE REVIEW

File: src/components/TransactionItem/TransactionItem.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  IconButton,
+  Tooltip,
+  useColorModeValue
+} from '@chakra-ui/react';
+import { format } from 'date-fns';
+
+import { NoteIcon } from '@/assets';
+
+export const TransactionItem = ({ transaction, onEdit, onDelete }) => {
+  const bgMain = useColorModeValue('orange.50', 'teal.700');
```
Comment (kalashnikau) [inline src/components/TransactionItem/TransactionItem.jsx:15]: use that variable from theme hook

File: src/pages/Landing.jsx [hunk 2]
```diff
@@ -33,6 +51,22 @@ export const Landing = () => {
         </Flex>
       </Box>
       <Link href="/wallet/123"> Draft of Wallet Page</Link>
+      <Box bg={bgMain} w="100%" p={4}>
+        <Flex bg={bgMain} direction="column" justify="center" align="center">
+          <Text fontSize="xl">
+            {i18next.t('transaction.recentTransactions')}
+          </Text>
+          <VStack w="80%" pt={5} spacing={5} align="stretch" justify="center">
+            {isFetched &&
```
Comment (kalashnikau) [inline src/pages/Landing.jsx:60]: create a component which will have name TransactionsList and will accept array of transactions and do that logic inside

**Review by kalashnikau — DISMISSED** [review]

**Review by LizavetaY — APPROVED** [review]

File: src/components/TransactionItem/TransactionItem.jsx (hunk 1 above)
Comment (lazzzaroni) [reply src/components/TransactionItem/TransactionItem.jsx:15]: You can take a look at Preload component if you wanna see some example how to use that hook

**Review by kalashnikau — APPROVED** [review]

File: src/components/TransactionList/TransactionList.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  IconButton,
+  Tooltip,
+  VStack
+} from '@chakra-ui/react';
+import { format } from 'date-fns';
+
+import { NoteIcon } from '@/assets';
+import { useCentralTheme } from '@/theme';
+
+export const TransactionList = ({ list }) => {
```
Comment (LizavetaY) [inline src/components/TransactionList/TransactionList.jsx:15]: Maybe we also need to get values about what kind of list we should do? I mean, accept an prop like 'Income' or 'Expense' or 'all' of them

**Review by LizavetaY — APPROVED** [review]

**Pull request merged**